### ISSUE TITLE
Testing report for EUR New and Old3DS (EUR crashing issues)

### ISSUE THREAD

**Issue by iCRON862** [zoogie/Kartminer7#1]
Hello, i have used time to testing the exploit in EUR n3ds and o3ds.

- Both wont works. Systems are on 11.17 and the game version is 1.2 as cartridge. It crashes after pressing 'Start Race'.

- the .bat says that i need to put in an hard drive in \device\harddisk2\dr2. I must close the window 3 times. Is this normal?

- If you set the language to english then you can't get past after open 'Streetpass List' because it crashes on the mii list.

**Comment by zoogie** [comment]
"\device\harddisk2\dr2" what in the world is this, lol.
Are you using Windows? 3ds wifi off?

**Comment by iCRON862** [comment]
> "\device\harddisk2\dr2" what in the world is this, lol.
> Are you using Windows? 3ds wifi off?

Hello, yeah i used windows 7, i used windows 10 too, same result 😅 wifi on console turned off.
![image](https://github.com/zoogie/Kartminer7/assets/39169140/855b8100-473d-4880-bf42-d9b432890471)
That's the message from the .bat in ger. You need an tranzlator to english 😅

**Comment by zoogie** [comment]
When the app crashes, does it go to a luma crash screen?
And what is your sd card's drive letter?

**Comment by iCRON862** [comment]
> When the app crashes, does it go to a luma crash screen?
> And what is your sd card's drive letter?

The system is crashed to normal home menu. The sd drive letter is F:

**Comment by zoogie** [comment]
Can you make a copy of your backup folder, and from that backup - copy, click on the backup_EUR.bat, the upload the 307 folder?
I just want to see what the system might have done to alter the save.

**Comment by iCRON862** [comment]
[307.zip](https://github.com/zoogie/Kartminer7/files/11946841/307.zip)
That's the exploited backup from Old. The backup-bat get the missing hard driver window too.

**Comment by zoogie** [comment]
As long as you get the 

Clearing the original contents...
Importing new contents...
"extdata/307\\boss"
"extdata/307\\boss\\wTRu2!!!(`!!+s(@"
"extdata/307\\icon"
"extdata/307\\user"
"extdata/307\\user\\data.dat"
Finished
Press any key to continue . . .

Then I suppose that nonsense warning is ok. Are you sure you downloaded that 307 folder from the 3ds and not the extdata folder from the Release_BETA archive?

**Comment by iCRON862** [comment]
> As long as you get the Clearing the original contents... Importing new contents... "extdata/307\boss" "extdata/307\boss\wTRu2!!!(`!!+s(@" "extdata/307\icon" "extdata/307\user" "extdata/307\user\data.dat" Finished Press any key to continue . . .
> 
> Then I suppose that nonsense warning is ok. Are you sure you downloaded that 307 folder from the 3ds and not the extdata folder from the Release_BETA archive?

I have installed the exploit on an clean o3ds to test. Then i go to my pc and maked an backup-dump from the exploit extdata used the backup-bat what created this 307 in 'backup' folder.

**Comment by zoogie** [comment]
Try this if you don't mind: change your region to 
United Kingdom / Scotland   (this is old3ds EUR only btw)
Confirm the language is English and try again.

I have a bad feeling this is super skaterhax all over again.

**Comment by iCRON862** [comment]
> Try this if you don't mind: change your region to United Kingdom / Scotland (this is old3ds EUR only btw) Confirm the language is English and try again.
> 
> I have a bad feeling this is super skaterhax all over again.

Not working, crashes after streetpass list button

**Comment by zoogie** [comment]
[debug_luma.zip](https://github.com/zoogie/Kartminer7/files/11947418/debug_luma.zip)
Could you run the exploit with that so I can get a crash dump? This is the last thing I'll ask today.
That particular luma doesn't have rosalina, so it can boot homebrew.

**Comment by iCRON862** [comment]
[crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11947511/crash_dumps.zip)

I have dumps from your english and one for the other languages settings. Have a nice day.

**Comment by zoogie** [comment]
Thank you very much for your help.

**Comment by CapnBloodBeard1** [comment]
Hello I have a problem in my old 3ds it’s USA and basically what happen when it asks “Start Race” and I click ok the screen just turn white and it crashes is there any way to fix this and if there is please help thank you so much

**Comment by zoogie** [comment]
@iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English?
[EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip)
It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120.
That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.

Thanks!

**Comment by iCRON862** [comment]
> @iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English? [EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip) It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120. That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.
> 
> Thanks!

Ok, i test it after work.

**Comment by iCRON862** [comment]
[crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11983441/crash_dumps.zip)

the ...000 is with Germany/ -- and ...001 is with your UK/ Scotland setting

**Comment by iCRON862** [comment]
> @iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English? [EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip) It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120. That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.
> 
> Thanks!

[crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11983464/crash_dumps.zip)

the ...000 is with Germany/ -- and ...001 is with your UK/ Scotland setting

**Comment by zoogie** [comment]
> > @iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English? [EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip) It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120. That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.
> > Thanks!
> 
> [crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11983464/crash_dumps.zip)
> 
> the ...000 is with Germany/ -- and ...001 is with your UK/ Scotland setting
Unfortunately, both of those crash dumps are corrupted, they both crash python.

Was English the set language on both? My research so far has shown that region/subregion don't matter, only lang matters. 
In any event, try to get crash dumps where the game reaches "start race: OK", and report which languages crash early by not making it past "Streetpass List", but I don't need .dmps from those.

**Comment by iCRON862** [comment]
> > > @iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English? [EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip) It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120. That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.
> > > Thanks!
> > 
> > 
> > [crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11983464/crash_dumps.zip)
> > the ...000 is with Germany/ -- and ...001 is with your UK/ Scotland setting
> > Unfortunately, both of those crash dumps are corrupted, they both crash python.
> 
> Was English the set language on both? My research so far has shown that region/subregion don't matter, only lang matters. In any event, try to get crash dumps where the game reaches "start race: OK", and report which languages crash early by not making it past "Streetpass List", but I don't need .dmps from those.

Yeah both was english
So for me, all other languages except English go through the StreetPass list up to 'OK'. English crashes on streetpass list.

**Comment by iCRON862** [comment]
> > > @iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English? [EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip) It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120. That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.
> > > Thanks!
> > 
> > 
> > [crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11983464/crash_dumps.zip)
> > the ...000 is with Germany/ -- and ...001 is with your UK/ Scotland setting
> > Unfortunately, both of those crash dumps are corrupted, they both crash python.
> 
> Was English the set language on both? My research so far has shown that region/subregion don't matter, only lang matters. In any event, try to get crash dumps where the game reaches "start race: OK", and report which languages crash early by not making it past "Streetpass List", but I don't need .dmps from those.

Yeah both are english.
So any language except english go past the streetpass list up to 'OK' with the same crash dump.
[crash_dump.zip](https://github.com/zoogie/Kartminer7/files/11985218/crash_dump.zip)

**Comment by zoogie** [comment]
> > > > @iCRON862 If you have some time to spare, could you try the this new test data on your old3ds EUR set to English? [EUR_test_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11979153/EUR_test_mk7v1.2.zip) It's intended for this to crash. The debug data I need will be displayed in r0 of the crash dump, ex. R0 0x44440120. That's all I need -- except if the R0 value doesn't start with 0x4444, then just upload the full .dmp file please.
> > > > Thanks!
> > > 
> > > 
> > > [crash_dumps.zip](https://github.com/zoogie/Kartminer7/files/11983464/crash_dumps.zip)
> > > the ...000 is with Germany/ -- and ...001 is with your UK/ Scotland setting
> > > Unfortunately, both of those crash dumps are corrupted, they both crash python.
> > 
> > 
> > Was English the set language on both? My research so far has shown that region/subregion don't matter, only lang matters. In any event, try to get crash dumps where the game reaches "start race: OK", and report which languages crash early by not making it past "Streetpass List", but I don't need .dmps from those.
> 
> Yeah both are english. So any language except english go past the streetpass list up to 'OK' with the same crash dump. [crash_dump.zip](https://github.com/zoogie/Kartminer7/files/11985218/crash_dump.zip)

Gah, zero bytes! Well, we'll try something different...

This will be the last thing today. Could you use that debug luma I posted above to make make an fcram dump?
This works by simply pressing START when you see the luma crash screen and then waiting about 15 seconds until the system turns off automatically. On the SD root will be a file, "fcram.bin". Just zip that up and upload it.

Again, only do this with a crash that gets to ""start race: OK". Thank you VERY much : ) I'm certain this file will shed some light on things.

**Comment by iCRON862** [comment]
No problem, 

> Gah, zero bytes! Well, we'll try something different...
> 
> This will be the last thing today. Could you use that debug luma I posted above to make make an fcram dump? This works by simply pressing START when you see the luma crash screen and then waiting about 15 seconds until the system turns off automatically. On the SD root will be a file, "fcram.bin". Just zip that up and upload it.
> 
> Again, only do this with a crash that gets to ""start race: OK". Thank you VERY much : ) I'm certain this file will shed some light on things.

No problem, i like testing exploits

(PS. Do you need the new3ds dump too?)

**Comment by zoogie** [comment]
> 

Just old3ds is fine for now, but you may need to redump or reupload that fcram file.

It's 0 bytes again, like the previous crash dumps. Is your sd card almost out of memory?

**Comment by iCRON862** [comment]
> > 
> 
> Just old3ds is fine for now, but you may need to redump or reupload that fcram file.
> 
> It's 0 bytes again, like the previous crash dumps. Is your sd card almost out of memory?

Oh, yeah. My sd space was full. Now the fcram is bigger as before. Now i hope these is complete
[fcram.zip](https://github.com/zoogie/Kartminer7/files/11992290/fcram.zip)

**Comment by zoogie** [comment]
With all those sd files showing up as 0 bytes I'd imagine extdata (where the exploit is installed) must have been corrupted too.

But anyway, here's something for you to try. Try to run it in whatever configuration your 3ds was in when you made the fcram dump.
[EUR_test2_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11995893/EUR_test2_mk7v1.2.zip)
Old or New 3ds but I'd start with old. They're combined now.
This is the first test of a major refactor.

**Comment by iCRON862** [comment]
Nothing changes, crashed after OK with the crash dump. Here is the actual fcram and dmp
[fcram.zip](https://github.com/zoogie/Kartminer7/files/12001672/fcram.zip)
[crash_dump_00000000.zip](https://github.com/zoogie/Kartminer7/files/12001682/crash_dump_00000000.zip)

**Comment by iCRON862** [comment]
> With all those sd files showing up as 0 bytes I'd imagine extdata (where the exploit is installed) must have been corrupted too.
> 
> But anyway, here's something for you to try. Try to run it in whatever configuration your 3ds was in when you made the fcram dump. [EUR_test2_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/11995893/EUR_test2_mk7v1.2.zip) Old or New 3ds but I'd start with old. They're combined now. This is the first test of a major refactor.

Nothing changes but here is the things you need
[fcram.zip](https://github.com/zoogie/Kartminer7/files/12001695/fcram.zip)
[crash_dump_00000000.zip](https://github.com/zoogie/Kartminer7/files/12001699/crash_dump_00000000.zip)

**Comment by zoogie** [comment]
Lets try this:
[EUR_test3_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12003086/EUR_test3_mk7v1.2.zip)
btw - if a crash occurs at streetpass list, don't send those. Can't even begin debugging them. 
Just send Start Race: OK crashes.

Thanks for your continued help.

**Comment by iCRON862** [comment]
> Lets try this: [EUR_test3_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12003086/EUR_test3_mk7v1.2.zip) btw - if a crash occurs at streetpass list, don't send those. Can't even begin debugging them. Just send Start Race: OK crashes.
> 
> Thanks for your continued help.

The only change i see is that R0 have 000000001 in the dmp. Here we go again. Yeah these get past streetpass list up to OK.
[crash_dump_00000001.zip](https://github.com/zoogie/Kartminer7/files/12003601/crash_dump_00000001.zip)
[fcram.zip](https://github.com/zoogie/Kartminer7/files/12003594/fcram.zip)

**Comment by zoogie** [comment]
> > Lets try this: [EUR_test3_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12003086/EUR_test3_mk7v1.2.zip) btw - if a crash occurs at streetpass list, don't send those. Can't even begin debugging them. Just send Start Race: OK crashes.
> > Thanks for your continued help.
> 
> The only change i see is that R0 have 000000001 in the dmp. Here we go again. Yeah these get past streetpass list up to OK. [crash_dump_00000001.zip](https://github.com/zoogie/Kartminer7/files/12003601/crash_dump_00000001.zip) [fcram.zip](https://github.com/zoogie/Kartminer7/files/12003594/fcram.zip)

That was quite different from the last one. Unfortunately, it looks even more unmanageable. 
The initial exploit context is completely lost.

I have 6 consoles from the 3 major regions and this works perfectly almost every time. Unless I can figure out what makes my consoles different from yours, I don't see how I can fix this.

I'll let you know if I can think of anything else to try.

**Comment by iCRON862** [comment]
> > > Lets try this: [EUR_test3_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12003086/EUR_test3_mk7v1.2.zip) btw - if a crash occurs at streetpass list, don't send those. Can't even begin debugging them. Just send Start Race: OK crashes.
> > > Thanks for your continued help.
> > 
> > 
> > The only change i see is that R0 have 000000001 in the dmp. Here we go again. Yeah these get past streetpass list up to OK. [crash_dump_00000001.zip](https://github.com/zoogie/Kartminer7/files/12003601/crash_dump_00000001.zip) [fcram.zip](https://github.com/zoogie/Kartminer7/files/12003594/fcram.zip)
> 
> That was quite different from the last one. Unfortunately, it looks even more unmanageable. The initial exploit context is completely lost.
> 
> I have 6 consoles from the 3 major regions and this works perfectly almost every time. Unless I can figure out what makes my consoles different from yours, I don't see how I can fix this.
> 
> I'll let you know if I can think of anything else to try.

What if you dumping your working exploit extdata with jksm and send it here that i can try to install this on my console to test it out?

**Comment by ThatRandomMaker** [comment]
One thing that i find is weird, that most of the time it crashes when you go to streetpass list, and sometimes you can start the race, but the game crashes then.

**Comment by zoogie** [comment]
Crashing at the streetpass list is very very bad and I can't even begin debugging it. Crashing at Start Race: OK is much closer to expected behavior.

**Comment by ThatRandomMaker** [comment]
Yeah, but it's kinda rng

**Comment by ThatRandomMaker** [comment]
Sometimes goes to "Start Race" sometimes doesn't.

**Comment by Devine3DS** [comment]
On my unhacked EUR 2DS i get to start race tab easily but then it mostly crashes with whitescreen and power off or a blackscreen and then it crashes

**Comment by ThatRandomMaker** [comment]
I've never got a black screen, only the crashing after start race

**Comment by Devine3DS** [comment]
Maybe zoogie can upload the latest version because im unable to compile it....

**Comment by ThatRandomMaker** [comment]
dunno

**Comment by ThatRandomMaker** [comment]
wait what ur talking about 1.2?

**Comment by ThatRandomMaker** [comment]
1.2 is already out

**Comment by Devine3DS** [comment]
Nah he updated the sourcecode 4 hours ago

**Comment by zoogie** [comment]
Testing time!
[EUR_test4_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12018042/EUR_test4_mk7v1.2.zip)
Set to English in system settings, try 2-3 times then go down the list and try different languages.
(remember to keep wifi off)

It's now showing Red -> Blue -> White on the bottom screen based on exploit stage. Any color Red and above is good, keep trying. Change language faster if you don't see any Red, Blue, etc.

**Comment by Devine3DS** [comment]
Is this for OLD or New 3DS?

**Comment by Devine3DS** [comment]
Okay i tried german spanish polish and italian. with the first 3 langs you come to the start race page but it crashs with a white screen with italien it freezes on the Mii

**Comment by Devine3DS** [comment]
Update with English language it goes red and stays on blue

**Comment by Devine3DS** [comment]
AND it worked!! i could install UnsafeMode!
[https://www.youtube.com/shorts/iD-MIjrWJsI](url)


Thank you for your awesome work!!

**Comment by zoogie** [comment]
Yeah when you see red or blue, you're really close. It's always good to keep trying when you see those colors, else change language.

Keep in mind this build is biased towards the offsets I saw in ICRON862's fcram dump, which happen to be a lot lower than mine. I think only English has a real chance with that address range for most people. I'm thinking about posting one that's more tailored to "typical" addresses here in a minute. Maybe others will have an easier time with it.

**Comment by Devine3DS** [comment]
I can test the Exploit on a few more devices if you want i got 2 more N3DS XL models and 1 2DS XL sadly my 2 old 3DS XLs passed away

**Comment by zoogie** [comment]
Yeah, if you have the time, that'd be great. Thanks!

I'd like to add a test5 here in a minute, that'd be a great starting point for those systems.

**Comment by Devine3DS** [comment]
Yes i got some spare time i can test but these systems all have cfw

**Comment by zoogie** [comment]
> Yes i got some spare time i can test but these systems all have cfw

Cfw is totally fine. If the exploit works perfectly, you'll get a Luma crash with "Current Process: menu". Failed attempts will crash on process "MarioKar".
The failed crash dumps can actually be really helpful (although it takes more time to upload them, so you don't have to).

**Comment by Devine3DS** [comment]
Okay i got a crash but i used the german language i will change it to english now on my n2DS XL
[arm11.zip](https://github.com/zoogie/Kartminer7/files/12021283/arm11.zip)


the first one was with german language the second one english on the old 2ds the exploit worked with english language

BTW does the 3DS ROP xPloit Injector Unsafemode backup the Saveslots? or do you need to remove the hacked wifi slots manually?

**Comment by zoogie** [comment]
It backs them up. Use the app slotTool.3dsx to restore the old wifi slots. It should be in the guide.

**Comment by Devine3DS** [comment]
Did you got any info off the logs?

**Comment by zoogie** [comment]
irl keeps intervening but yes, very helpful. 
now trying to get a test5 out but I keep adding stuff that I forgot about.

**Comment by zoogie** [comment]
I think we're closing in on a release candidate. Just a little farther to go.
Test5 is here now.
[EUR_test5_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12024036/EUR_test5_mk7v1.2.zip)

**Comment by Devine3DS** [comment]
Okay ive tested it on the N2DS XL and it booted into 3DS ROP xPloit Injector?? With CFW no Crash first try

![432432](https://github.com/zoogie/Kartminer7/assets/139223121/127fda3c-34b2-4c20-a447-da3f9e3e48d0)


If you press Start Button it will crash Exception type: prefetch abort (svcBreak) Arm9



On my OLD 2DS it might works too (Current process: menu) First try with German Language Crashlog is here 
[crash_dump_00000000.zip](https://github.com/zoogie/Kartminer7/files/12024127/crash_dump_00000000.zip)


On my 1 N3DS XL the 3DS ROP xPloit Injector boots directly! 

![2352325](https://github.com/zoogie/Kartminer7/assets/139223121/63fd52b5-cce1-40d8-93a0-25885cb121b5)


![234432423](https://github.com/zoogie/Kartminer7/assets/139223121/602144dd-046e-4012-9f04-bc111f904293)

and same error as on N2DS XL when exiting with Start


And my last N3DS XL with CFW german Language:

It just boots directly into the 3DS ROP xPloit Injector!

Its working on all my Consoles!

**Comment by ThatRandomMaker** [comment]
What happened? Is there an update or something?

**Comment by ThatRandomMaker** [comment]
oh, ill try test4

**Comment by ThatRandomMaker** [comment]
Still getting a lot of crashes on o2ds though

**Comment by Devine3DS** [comment]
Which Language did you set?

**Comment by ThatRandomMaker** [comment]
english

**Comment by Devine3DS** [comment]
Okay let me try. Ive uninstalled Boot9Strap on my O2DS. German Language is instant Boot no Crash.

I will try english

**Comment by ThatRandomMaker** [comment]
ok

**Comment by ThatRandomMaker** [comment]
since im still trying now but its just crashing

**Comment by ThatRandomMaker** [comment]
(im using test4 version btw)

**Comment by Devine3DS** [comment]
Okay just booted fine with English Language. You should use Test5 he fixed alot in it!

**Comment by ThatRandomMaker** [comment]
ok but

**Comment by ThatRandomMaker** [comment]
how do i unspoof my data now?

**Comment by ThatRandomMaker** [comment]
cuz the testv4 didn't have backup

**Comment by Devine3DS** [comment]
You dont need to just overwrite with the Test5 you can later restore with the original Restore.bat

**Comment by ThatRandomMaker** [comment]
No i mean like i deleted the old backup

**Comment by ThatRandomMaker** [comment]
because i thought it was gonna be fine with test4

**Comment by Devine3DS** [comment]
You dont need it just delete the 00000307 in your extdata then its clear again and generates new if you click Channel

**Comment by ThatRandomMaker** [comment]
ok, thanks! ill try it later since i need to go eat breakfast atm

**Comment by zoogie** [comment]
> because i thought it was gonna be fine with test4

Test5 is the latest if you haven't tried it.
[EUR_test5_mk7v1.2.zip](https://github.com/zoogie/Kartminer7/files/12024036/EUR_test5_mk7v1.2.zip)

If you keep getting crashes with no Red/Blue bottom screens, move on to a different language in System Settings and try again.
If you do get Red/Blue screens, keep trying on that language and you'll eventually get it.

**Comment by Devine3DS** [comment]
My Consoles now are running perfectly with the Exploit! Tried on unmodified 2DS with several Languages no Crashes so far

**Comment by ThatRandomMaker** [comment]
Btw "IF" i get into the injector what do i do? Since i've never homebrewed a 3ds

**Comment by Devine3DS** [comment]
Press X and follow this guide https://3ds.hacks.guide/installing-boot9strap-(usm) but Skip Section II - BannerBomb3 entirely

**Comment by ThatRandomMaker** [comment]
Will it still take a few tries to get into the start race screen?

**Comment by Devine3DS** [comment]
No i dont had any problems just press a on the Mii dont scroll or something it will crash.

**Comment by ThatRandomMaker** [comment]
hm, weird?

**Comment by ThatRandomMaker** [comment]
i'm still getting crashes when i press streetpass list

**Comment by ThatRandomMaker** [comment]
Should i try changing the language?

**Comment by Devine3DS** [comment]
Yes try it did you press slowly a?

**Comment by ThatRandomMaker** [comment]
Yes, im slowly pressing a

**Comment by ThatRandomMaker** [comment]
I've tried it normally, slow, fast

**Comment by ThatRandomMaker** [comment]
ill try changing the language to russian since i kinda know it

**Comment by Devine3DS** [comment]
Try chaning the Language

**Comment by ThatRandomMaker** [comment]
ok i got to start race

**Comment by ThatRandomMaker** [comment]
white screen + crash

**Comment by Devine3DS** [comment]
Okay then another Language

**Comment by ThatRandomMaker** [comment]
ill try it just one more time

**Comment by ThatRandomMaker** [comment]
yeah, the same but it gets to streetpass list very well

**Comment by Devine3DS** [comment]
Try Spanish or German

**Comment by ThatRandomMaker** [comment]
i don't have german as an option, probably cuz of my region set to Latvia

**Comment by ThatRandomMaker** [comment]
YES I GOT RED, THEN BLUE AND IM NOW IN INJECTOR

**Comment by ThatRandomMaker** [comment]
ok so what do i do now

**Comment by Devine3DS** [comment]
Which language did you use

**Comment by ThatRandomMaker** [comment]
espanol

**Comment by ThatRandomMaker** [comment]
ok but what do i do

**Comment by ThatRandomMaker** [comment]
i press x to install unsafe mode?

**Comment by Devine3DS** [comment]
yes maybe write me your discord i can help you if its your first time

**Comment by ThatRandomMaker** [comment]
oops i didn't have all of the files i needed on the sd card

**Comment by ThatRandomMaker** [comment]
ok

**Comment by ThatRandomMaker** [comment]
wait send me ur dc

**Comment by Devine3DS** [comment]
zazzcode

**Comment by ThatRandomMaker** [comment]
sent

**Comment by ThatRandomMaker** [comment]
It worked, changed language to spanish, first try!

**Comment by zoogie** [comment]
https://github.com/zoogie/Kartminer7/releases/tag/v0.2beta

**Comment by Devine3DS** [comment]
Did you change something to last Test5?

**Comment by zoogie** [comment]
> Did you change something to last Test5?

Very small changes but hoping there were no regressions.

**Comment by THZoria** [comment]
Test with French language
It works

**Comment by Devine3DS** [comment]
I tried it today on an Old 3DS launch model and worked first time. I tried to help a guy from UK with his Old 3DS XL and nothing worked with English language it crashed on the driver with other languages just whitescreens and crash. Tried every language and tried your alternative version with no luck.
